probert/_nl80211module.c

```diff
@@ -420,6 +420,18 @@ static void extract_ssid(struct nlattr *data, struct scan_handler_params *p)
 	char *ie = nla_data(bss[NL80211_BSS_INFORMATION_ELEMENTS]);
 	size_t ie_len = nla_len(bss[NL80211_BSS_INFORMATION_ELEMENTS]);
 	char *ssid = nl80211_get_ie(ie, ie_len, 0);
+
+	if (ssid == NULL) {
+		/*
+		 * LP: #2104087 For reasons yet to be determined, the SSID information
+		 * element (aka. IE) can sometimes be completely missing.
+		 * We have speculated that it could be related to hidden SSIDs but
+		 * testing showed that having an SSID information element with size 0
+		 * is a thing.
+		 */
+		return;
```

Comment:
non-blocking: similar to Dan's comment but maybe not exactly related, what's the expected result here where we early return and don't do any `PyObject` setup stuff? It seems similar enough to the other early returns but TBH I don't fully understand the expectations there either.

Reply:
My intention is to skip the current network (i.e., not adding to the list using Py_ListAppend()) but continue looking at other networks.